design/providerid_format_change.md

```diff
@@ -0,0 +1,154 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# providerid_format_change
+
+## Status
+
+provisional
+
+## Summary
+
+The providerID is generated using Baremetalhost's uuid.
+While the Baremetalhost's uuid is tied to the life of the BMH object,
+the providerID should be related to a provisioning a kubernetes cluster.
+
+The current providerID has some limitations due to content and format.
+The providerID uses Baremetalhost's uuid, making it static for the life of
+the BMH.
```

Comment:
This essentially makes a providerID tied to a Baremetal hosts's lifecycle which is sometimes undesirable.